build/common.sh

```diff
@@ -573,23 +542,47 @@ function kube::build::ensure_data_container() {
     # We want this to run as root to be able to chown, so non-root users can
     # later use the result as a data container.  This run both creates the data
     # container and chowns the GOPATH.
+    #
+    # The data container creates volumes for all of the directories that store
```

Comment:
Not really -- subtleties here:
* The way that host directories are shared into the VM and bind mounted into the container is super slow through VirtualBox.  This is one of the big things that comes from Docker for Mac
* Not all of the intermediate files are under _output.  The fact that we need to recompile the standard libraries (and if we don't cache the world gets rebuilt) means we need to map /usr/local/go/pkg/linux_amd64_cgo too.

If we knew that bind mounts would be fast enough, perhaps we could make it work.  But they are *really* slow with VirtualBox.